backend/src/Squidex.Infrastructure/Queries/Query.cs

```diff
@@ -24,6 +24,8 @@ public long Top
             set { Take = value; }
         }
 
+        public RefToken? CreatedBy { get; set; }
```

Comment:
Please do not add it here.

Reply:
I use it in QueryByQuery - CreateFilter

```
if (query?.CreatedBy != null)
            {
                filters.Add(Filter.Eq(x => x.CreatedBy, query.CreatedBy));
            }
```

How better than implement it?

Reply:
I mean don't add it to Query, only to Q

Reply:
If I remove this property, how should I rewrite this logic?

```
if (query?.CreatedBy != null)
            {
                filters.Add(Filter.Eq(x => x.CreatedBy, query.CreatedBy));
            }
```

Reply:
Just take it from Q